### ISSUE TITLE
Problems about how to fine-tuning

### ISSUE THREAD

**Issue by p994070003** [VITA-Group/EnlightenGAN#125]
Hello Sir.

Thanks for releasing this amazing implementation, the results are great!

I found no the code about how to fine tune the modle. I wanna continue to train the model with my dark datasets in trainA and the dataset that you provided in trainB  by a smaller learning rate,How should I operate it?

Any help will be appreciated!

Thanks.
Nova.

**Comment by jiejiangd** [comment]
想请问一下这个可以单卡训练吗

**Comment by yueruchen** [comment]
finetuning is not supported, you might need to change the code a little bit. E.g., loading the pretrained weight before starting your training job. @p994070003

**Comment by yueruchen** [comment]
@jiejiangd 
single-gpu is doable but not guaranteed to reproduce it
